jenkins-pipelines/performance/branch-perf-v15/scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-scylla-gradual-throughput-grow-2.3gb.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_gradual_grow_throughput.PerformanceRegressionGradualGrowThroughputTest",
+    test_config: '''["test-cases/performance/perf-regression-gradual-throughput-grow-2.3tb.yaml", "configurations/performance/cassandra_stress_gradual_load_steps_enterprise.yaml", "configurations/disable_kms.yaml", "configurations/tablets_disabled.yaml", "configurations/disable_speculative_retry.yaml"]''',
+    sub_tests: ["test_write_gradual_increase_load", "test_read_gradual_increase_load", "test_mixed_gradual_increase_load"],
+
+    timeout: [time: 1600, unit: "MINUTES"]
```

Comment:
we don't need timeout in those pipelines, it's not doing anything
I'll remove it